src/ScheduleFunctions.cpp

```diff
@@ -995,26 +1000,31 @@ void validate_schedule(Function f, Stmt s, const Target &target, bool is_output,
     // store_in_user_code, but store_root is close enough.
     if (is_output) {
         if (store_at.is_root() && compute_at.is_root()) {
-            return;
+            return true;
         } else {
             user_error << "Func " << f.name() << " is the output, so must"
```

Comment:
Might be worth changing "the output" to "an output" now that we support multiple outputs.